test.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Cluster
```

Comment:
Could you please remove this file? It’s useless, isn't it?

Reply:
Oops - yeah did not mean to commit that one